hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1463,15 +1463,15 @@ possible configurations would overwhelm and obscure the important.
   </property>
   <property>
     <name>dfs.client.read.shortcircuit</name>
-    <value>false</value>
+    <value></value>
     <description>
       If set to true, this configuration parameter enables short-circuit local
       reads.
     </description>
   </property>
   <property>
     <name>dfs.domain.socket.path</name>
-    <value>none</value>
+    <value></value>
```

Comment:
I don't think this works either. I think we need to not have this property at all. I think this empty config property would still mask the real one, right?

Reply:
It is reserved to provide description.This empty config property would not mask the real one.You can verify this from unit tests.

Reply:
`assertNull(conf.get("dfs.domain.socket.path"));`
If "hbase - default. XML configuration in effect, there will be an error, like this: 
Java. Lang. AssertionError: expected null, but the was: < none >

Reply:
`assertEquals("/var/lib/hadoop-hdfs/dn_socket", conf.get("dfs.domain.socket.path"));`
If this empty config property would still mask the real one, there will be an error.